src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpPreferDictionaryTryGetValueFixer.cs

```diff
@@ -35,6 +36,13 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                 return;
             }
 
+            var model = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
+            if (model.GetTypeInfo(dictionaryAccess).Type is not { } type)
+            {
+                return;
```

Comment:
Not sure if the type is always available, probably should use `var` if it is not